README.md

```diff
@@ -7,3 +7,8 @@ number as part of the name.
 
 To use the files download the archive and extract them to get the test files and readme.
 
+October 28 2017
+---------------
+
+Added a directory simpleTestForGenerator containing test files for the SPDX License Generator.  These files should be close to the canonical text for the listed license files.
```

Comment:
I'd rather drop “close to”.  They should *be* the text form (for `.txt` files) of our canonical text.

Also, why `simpleTestForGenerator`?  If the values are the canonical license text, then why not name the directory `canonical`?  The paths should encode what the data *is*, not what we expect it to be used for.

And the `{use-case}/{license}.{extention}` approach inverts the scheme I proposed in #3.  I'd rather keep the license as the first element and use `{license}/license/match/canonical.txt` for these.